### PR TITLE
[NCL-6167] Use new PME/DA endpoint parameters

### PR DESCRIPTION
**Pull request by thescouser89**
The new PME (4.3) and DA (2.x) introduces 2 new parameters:

- brewPullActive (default: false)
- mode (default: "")

The 'brewPullActive' flag is set on whether to also consider artifacts
from Brew during alignment.

The mode is used to specify to which aligned versions to align to. Currently
there are 4 modes:

- PERSISTENT (permanent aligned builds)
- TEMPORARY (temporary aligned builds)
- SERVICE (service aligned builds)
- SERVICE-TEMPORARY (temporary service aligned builds)

Those modes and the corresponding aligned version suffixes are
configured on the Dependency Analysis service.

### CODE REVIEW

**Comment by rnc** [comment]
@thescouser89 Thanks! I know this is draft :-) but once done we'll need an update to the gh-pages branch as well I think

File: build.gradle.kts [hunk 1]
```diff
@@ -129,7 +129,7 @@ subprojects {
     extra["logbackVersion"] = "1.2.3"
     extra["mavenVersion"] = "3.5.0"
     extra["ownerVersion"] = "1.0.12"
-    extra["pmeVersion"] = "4.2"
+    extra["pmeVersion"] = "4.3-SNAPSHOT"
```
Comment (rnc) [inline build.gradle.kts:132]: This can be updated to 4.3 now please

**Comment by rnc** [comment]
@thescouser89 As part of the move to 4.3 could you remove the `TODO` from https://github.com/project-ncl/gradle-manipulator/blob/master/common/src/main/java/org/jboss/gm/common/groovy/BaseScript.java#L107 please?

**Comment by thescouser89** [comment]
Aye codecov isn't happy. Let me know if you want me to add more tests or something.

**Review by rnc — APPROVED** [review]

**Pull request merged**